### DISCUSSION TITLE
Fetching Ordinals from an Address

### DISCUSSION THREAD

**Discussion by jeffery-jurat** [ordinals/ord#2162] [Q&A]
Hi, I wanted to ask if there was a function to allow viewing of all ordinals connected/minted from an address

**Comment by literalmonkey** [comment]
there is not afaik. that would require a new end point.

**Comment by khanzada-web** [comment]
can i get an api how to fetch all the ordinals from wallet

**Reply by raphjaph** [reply]
you can fetch all outpoints for and address from `/address/<ADDRESS>` and then fetch all inscriptions, runes, etc. for an outpoint from `/output/<OUTPOINT>
